rotator_indi.cpp

```diff
@@ -0,0 +1,328 @@
+/*
+*  rotator_indi.cpp
+*  PHD Guiding
+*
+*  Created by Philipp Weber
+*  Copyright (c) 2023 Philipp Weber
+*  Copyright (c) 2024 Kirill M. Skorobogatov
```

Comment:
https://github.com/OpenPHDGuiding/phd2/pull/1091#issuecomment-1979760396
```suggestion
*  Created by Philipp Weber and Kirill M. Skorobogatov
*  Copyright (c) 2024 openphdguiding.org
```